### PR TITLE
create one team and show all teams (teams index)

### PR DESCRIPTION
**Pull request by PhuongTrieu**

### CODE REVIEW

File: app/controllers/admin/teams_controller.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class Admin::TeamsController < ApplicationController
+  before_action :admin_user
+
+  def index
+    @teams = Team.paginate page: params[:page], per_page:
+                                  Settings.number_per_page
```
Comment (brokenheart1712) [inline app/controllers/admin/teams_controller.rb]: Xem lại indent em ơi.

File: app/controllers/admin/teams_controller.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  end
+
+  def new
+    @team = Team.new
+  end
+
+  def create
+    @team = Team.new team_params
+    if @team.save
+      flash[:succes] = t "team.created"
```
Comment (brokenheart1712) [inline app/controllers/admin/teams_controller.rb]: t :team.created

File: app/views/admin/skills/_form.html.erb [hunk 3]
```diff
@@ -1,8 +1,8 @@
 <%= form_for [:admin, @skill] do |f| %>
-  <%= render "shared/skill_error_messages" %>
+  <%= render 'shared/error_messages', object: f.object %>
```
Comment (brokenheart1712) [inline app/views/admin/skills/_form.html.erb]: Dùng " là đúng rồi, sao lại phải đổi về '

File: app/views/admin/teams/_form.html.erb [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+  <%= render 'shared/error_messages', object: f.object %>
+  <div class="fields">
+    <%= f.label :name %>
+    <%= f.text_field :name, class: "form-control" %><br>
+    <%= f.label :description %>
+    <%= f.text_area :description, rows: 4 %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit t("submit"), class: "btn btn-primary" %>
```
Comment (brokenheart1712) [inline app/views/admin/teams/_form.html.erb]: t(:submit)

File: app/views/admin/teams/_team.html.erb [hunk 5]
```diff
@@ -0,0 +1,10 @@
+<tr>
+  <td><%= link_to team.name, admin_team_path(team) %></td>
+  <td><%= team.created_at %></td>
+  <td><%= link_to "", edit_admin_team_path(team),
```
Comment (brokenheart1712) [inline app/views/admin/teams/_team.html.erb]: Nên có nhãn cho link_to, chẳng hạn link_to "Edit Team As Admin".

File: app/views/admin/teams/_team.html.erb [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+              class: "glyphicon glyphicon-edit" %>
+      <%= link_to "", admin_team_path(team), method: :delete,
```
Comment (brokenheart1712) [inline app/views/admin/teams/_team.html.erb]: Nên có nhãn cho link_to, chẳng hạn link_to "Edit Team As Admin".

File: app/views/admin/teams/index.html.erb [hunk 7]
```diff
@@ -0,0 +1,24 @@
+<p id="notice"><%= notice %></p>
+
+<h1><%= t "team.list" %></h1>
+
+<button >
+<%= link_to t("team.new"), new_admin_team_path,
```
Comment (brokenheart1712) [inline app/views/admin/teams/index.html.erb]: t(:team.new)

File: app/views/admin/teams/new.html.erb [hunk 8]
```diff
@@ -0,0 +1,8 @@
+<h1><%= t "team.new" %></h1>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= render "form" %>
+
+    <br><%= link_to t("back"), admin_teams_path %>
```
Comment (brokenheart1712) [inline app/views/admin/teams/new.html.erb]: t(:back)

File: app/views/layouts/_header.html.erb [hunk 9]
```diff
@@ -10,10 +10,12 @@
             </a>
             <ul class="dropdown-menu">
               <li><%= link_to t("user"), "#" %></li>
```
Comment (brokenheart1712) [inline app/views/layouts/_header.html.erb]: t(:user)

File: app/views/shared/_error_messages.html.erb [hunk 10]
```diff
@@ -0,0 +1,12 @@
+<% if object.errors.any? %>
+  <div id="error_explanation">
+    <div class="alert alert-danger">
+      The form contains <%= pluralize(object.errors.count, "error") %>.
+    </div>
+    <ul>
+    <% object.errors.full_messages.each do |msg| %>
```
Comment (brokenheart1712) [inline app/views/shared/_error_messages.html.erb]: Indent

File: app/controllers/admin/teams_controller.rb [hunk 11]
```diff
@@ -0,0 +1,27 @@
+class Admin::TeamsController < ApplicationController
+  before_action :admin_user
+
+  def index
+    @teams = Team.paginate page: params[:page], per_page:
+    Settings.number_per_page
```
Comment (vjnhat) [inline app/controllers/admin/teams_controller.rb]: indent

Comment (khanhkhanh) [inline app/controllers/admin/teams_controller.rb]: @teams = Team.paginate page: params[:page], per_page: Settings.number_per_page
or
@teams = Team.paginate page: params[:page],
                                          per_page: Settings.number_per_page

File: app/views/admin/teams/index.html.erb (hunk 7 above)
Comment (vjnhat) [inline app/views/admin/teams/index.html.erb]: indent

File: app/views/admin/skills/index.html.erb [hunk 12]
```diff
@@ -2,9 +2,9 @@
 
 <h1><%= t "skill.list" %></h1>
 
-<button >
+<button>
 <%= link_to t("skill.new"), new_admin_skill_path,
-                          class: "glyphicon glyphicon-plus" %>
+  class: "glyphicon glyphicon-plus" %>
```
Comment (vjnhat) [inline app/views/admin/skills/index.html.erb]: indent

File: app/views/admin/teams/index.html.erb [hunk 13]
```diff
@@ -0,0 +1,24 @@
+<p id="notice"><%= notice %></p>
+
+<h1><%= t "team.list" %></h1>
+
+<button>
+<%= link_to t("team.new"), new_admin_team_path,
```
Comment (vjnhat) [inline app/views/admin/teams/index.html.erb]: indent

File: app/views/admin/teams/index.html.erb [hunk 14]
```diff
@@ -0,0 +1,24 @@
...
+  class: "glyphicon glyphicon-plus" %>
+</button>
+<%= will_paginate %>
+<div class="table-responsive">
+  <table class="table">
+    <thead>
+      <tr>
+        <th>Name</th>
```
Comment (vjnhat) [inline app/views/admin/teams/index.html.erb]: i18n

File: app/views/shared/_error_messages.html.erb [hunk 15]
```diff
@@ -0,0 +1,14 @@
+<% if object.errors.any? %>
+  <div id="error_explanation">
+    <div class="alert alert-danger">
+      The form contains <%= pluralize(object.errors.count, "error") %>.
```
Comment (khanhkhanh) [inline app/views/shared/_error_messages.html.erb]: i18n

File: app/views/admin/skills/index.html.erb [hunk 16]
```diff
@@ -2,9 +2,9 @@
 
 <h1><%= t "skill.list" %></h1>
 
-<button >
+<button>
 <%= link_to t("skill.new"), new_admin_skill_path,
```
Comment (vjnhat) [inline app/views/admin/skills/index.html.erb]: indent

Comment (vjnhat) [inline app/views/admin/skills/index.html.erb]: indent

File: app/views/admin/teams/index.html.erb [hunk 17]
```diff
@@ -0,0 +1,21 @@
+<h1><%= t "team.list" %></h1>
+<button>
+<%= link_to t("team.new"), new_admin_team_path,
```
Comment (vjnhat) [inline app/views/admin/teams/index.html.erb]: indent

File: app/views/admin/skills/index.html.erb [hunk 18]
```diff
@@ -2,9 +2,9 @@
 
 <h1><%= t "skill.list" %></h1>
 
-<button >
+<button>
```
Comment (khanhkhanh) [inline app/views/admin/skills/index.html.erb]: Remove button tags.
Using glyphicon and glyphicon-plus is enough

File: app/views/shared/_error_messages.html.erb [hunk 19]
```diff
@@ -0,0 +1,14 @@
+<% if object.errors.any? %>
+  <div id="error_explanation">
+    <div class="alert alert-danger">
+      <%= t(:error_alert) %><%= pluralize(object.errors.count, "error") %>.
```
Comment (khanhkhanh) [inline app/views/shared/_error_messages.html.erb]: First, you have to remove ()s, and this line will be
<%= t :error_alert %><%= pluralize object.errors.count, "error" %>
Then you change the second part to <%= pluralize object.errors.count, t(:error) %>
After that, define error inside i18n file :)

File: app/controllers/admin/skills_controller.rb [hunk 20]
```diff
@@ -3,7 +3,7 @@ class Admin::SkillsController < ApplicationController
 
   def index
     @skills = Skill.paginate page: params[:page], per_page:
-    Settings.number_per_page
+                            Settings.number_per_page
```
Comment (khanhkhanh) [inline app/controllers/admin/skills_controller.rb]: Move per_page to the second line

File: app/controllers/admin/teams_controller.rb [hunk 21]
```diff
@@ -0,0 +1,27 @@
+class Admin::TeamsController < ApplicationController
+  before_action :admin_user
+
+  def index
+    @teams = Team.paginate page: params[:page], per_page:
+                          Settings.number_per_page
```
Comment (khanhkhanh) [inline app/controllers/admin/teams_controller.rb]: Move per_page to the second line

File: app/views/admin/skills/index.html.erb [hunk 22]
```diff
@@ -1,19 +1,15 @@
...
+  class: "glyphicon glyphicon-plus" %>
 <%= will_paginate %>
 <div class="table-responsive">
   <table class="table">
     <thead>
       <tr>
-        <th>Name</th>
-        <th>Date time</th>
-        <th>Modified</th>
+        <th><%= t (:name) %></th>
```
Comment (khanhkhanh) [inline app/views/admin/skills/index.html.erb]: remove ()s

File: app/views/admin/teams/index.html.erb [hunk 23]
```diff
@@ -0,0 +1,20 @@
+<h1><%= t "team.list" %></h1>
+
+<%= link_to t("team.new"), new_admin_team_path,
+  class: "glyphicon glyphicon-plus" %>
+<%= will_paginate %>
+<div class="table-responsive">
+  <table class="table">
+    <thead>
+      <tr>
+        <th><%= t (:name) %></th>
```
Comment (khanhkhanh) [inline app/views/admin/teams/index.html.erb]: Remove ()s

File: app/controllers/admin/skills_controller.rb [hunk 24]
```diff
@@ -2,8 +2,8 @@ class Admin::SkillsController < ApplicationController
   before_action :admin_user
 
   def index
-    @skills = Skill.paginate page: params[:page], per_page:
-    Settings.number_per_page
+    @skills = Skill.paginate page: params[:page],
+                            per_page: Settings.number_per_page
```
Comment (khanhkhanh) [inline app/controllers/admin/skills_controller.rb]: "per_page" should have the same indentation as "page"

File: app/controllers/admin/teams_controller.rb [hunk 25]
```diff
@@ -0,0 +1,27 @@
+class Admin::TeamsController < ApplicationController
+  before_action :admin_user
+
+  def index
+    @teams = Team.paginate page: params[:page],
+                          per_page: Settings.number_per_page
```
Comment (khanhkhanh) [inline app/controllers/admin/teams_controller.rb]: "per_page" should have the same indentation as "page"

**Pull request merged**